assets/scss/base.scss

```diff
@@ -47,14 +52,14 @@ body {
 }
 
 html {
-  font-size: 16px;
+  font-size: $px-16;
```

Comment:
This makes it easier to see/find what font sizes are used, and should help reduce errors, too.